modules/yardstick/README.txt

```diff
@@ -2,46 +2,93 @@ Yardstick Ignite Benchmarks
 ===========================
 Yardstick Ignite is a set of Ignite Grid (http://ignite.apache.org/) benchmarks written on top of Yardstick framework.
 
-Yardstick Framework
-===================
-Visit Yardstick Repository (https://github.com/gridgain/yardstick) for detailed information on how to run Yardstick benchmarks and how to generate graphs.
 
-The documentation below describes configuration parameters in addition to standard Yardstick parameters.
+Building the project
+====================
+Please look for the building instructions in `DEVNOTES.txt`
+
+
+Running Ignite Benchmarks
+=========================
+
+The simplest way to run benchmarks is running following command in `benchmarks` directory:
+
+./bin/benchmark-run-all.sh config/benchmark-atomic-put.properties
+
+Without any additional changes this command will run the PutBenchmark on your local machine. By default it takes
+1 minute to warm-up and 5 minutes to run a benchmark, so after 6 minutes the collected results should be in
+`results-DATE-TIME` directory.
+NOTE: You need to configure ssh key-based authentication to localhost for running benchmarks on your local machine.
+
+For running benchmarks on remote hosts you need to upload Ignite-Yardstick to each one of your remote hosts.
+
+NOTE: The path to the uploaded Ignite-Yardstick should be exactly the same on each host.
+
+Then you need to make some changes in config files:
 
-Installation
-============
-1. Create a local clone of Ignite repository
-2. Run
+1. You need to comment or delete the 30th
```

Comment:
Don't specify line numbers. It's subject to change.

Basically, the section can be split on two parts: for those who plan to run benchmarks locally and remotely. This will help to avoid any kind of confusion.